stories/LogoSubtitleBody.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { storiesOf, addDecorator } from '@storybook/react'
+import Theme from './theme'
+import { Grid, Col, Row } from '../src/components/grid'
+import WhyOpenSource from '../src/components/OpenSource/WhyOpenSource'
+
+import icon from './assets/community_support.svg'
+
+addDecorator(Theme)
+
+const title = 'Community support'
+const body =
+  'Open source technologies are supported by knowledgeable and resourceful communities. Their collective contributions result in more robust, innovative and faster solutions.'
+
+storiesOf('Logo, Subtitle & Body', module).add('on white Background', () => {
+  return (
+    <Grid>
+      <Row>
+        <Col width={[1, 1, 1, 1, 0.5]}>
```

Comment:
I would leave `Grid`, `Row` & `Col` out of the `LogoSubtitleBody` story because this small component may be re-used in other contexts where there are a different number of columns.

You would need `Grid`, `Row` & `Col` for the overall WhyOpenSource section though :+1: